File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BRIDGE/bridge_command.dart
import 'package:intent_digital_hub/IntentDigitalHubService/intent_digital_hub_command_starter.dart';
import 'package:intent_digital_hub/IntentDigitalHubService/intent_digital_hub_module.dart';

abstract class BridgeCommand extends IntentDigitalHubCommand {
  BridgeCommand(String functionName)
      : super(functionName, IntentDigitalHubModule.BRIDGE);
}

File: Exemplos M10/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Bridge/BridgeCommand.cs
using Newtonsoft.Json;

namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Bridge
{
    /**
     * Classe que generaliza todos os comandos do módulo BRIDGE, definindo o módulo do comando e o seu tipo de retorno
     */
    abstract class BridgeCommand : IntentDigitalHubCommand
    {
        //O retorno dos comandos bridge é sempre um JSON em String
        [JsonProperty(PropertyName = "resultado")]
        private readonly string resultado;

        protected BridgeCommand(string functionName) : base(functionName, IntentDigitalHubModule.BRIDGE) { }

        public string GetResultado()
        {
            return resultado;
        }
    }
}